share/instruments/instruments.xml

```diff
@@ -2983,8 +2983,8 @@
             </Instrument>
             <Instrument id="d-horn">
                   <longName>Horn in D</longName>
-                  <shortName>Hn.</shortName>
-                  <description>Horn</description>
+                  <shortName>D Hn.</shortName>
+                  <description>D Horn</description>
```

Comment:
Any reason for calling one "X Horn" and the other "Horn in X" (where X is the key)? Same goes for the short names. Why not use the same format everywhere? (I personally prefer "Instrument in X" over "X Instrument".)

Reply:
This is just in the Description tag, which is not used anywhere

Reply:
My mistake. I see that both use the following form:

```xml
<longName>Horn in X</longName>
<shortName>X Hn.</shortName>
```

I was confused by the `<description>`, and by the fact that you edited the `<shortName>` in one and the `<longName>` in the other.

Names are now consistent between Instruments, but I wonder whether the ordering should also be consistent between long and short name?

```xml
<longName>Horn in X</longName>
<shortName>Hn. in X</shortName>
````

Reply:
That'd make the short name 3 letters longer and as such defeata the purpose

Reply:
I like to be find instruments quickly just by glancing at the first letter, and I personally am willing to put up with a few extra characters to retain that ability. In any case, the solution recommened in Behind Bars allows you to get the best of both world by putting the key on a second line:

Hn.
in X

It looks a bit odd here, but it actually works very well in context (see [this score](https://musescore.com/openscore/scores/5271023), for example).